docs/CONTRIBUTING.md

```diff
@@ -55,22 +57,19 @@ To contribute code to ODK Collect, you will need to open a [pull request](https:
 
 1. Keep your pull request focused on one narrow goal. This could mean addressing an issue with multiple, smaller pull requests. Small pull requests are easier to review and less likely to introduce bugs. If you would like to make stylistic changes to the code, create a separate pull request.
 
-1. Run `./gradlew checkCode` and fix any errors.
-
 1. Write clear code. Use descriptive names and create meaningful abstractions (methods, classes).
 
 1. Document your reasoning. Your commit messages should make it clear why each change has been made.
 
 1. Point out decisions you made and what alternatives you considered. If you're unsure about a particular approach, ask a question to make your own thinking clear and help the reviewer identify controversial parts of the proposed solution. For example: "here I returned a result object to represent the status after the transaction. I also considered throwing an exception in case of error but I didn't like that it made it unclear where an error happened. Which do you prefer and why?" This is particularly important for [work in progress pull requests](#work-in-progress-pull-requests).
 
-1. If your pull request makes user-facing changes, we likely need to update documentation. [File an issue on the docs repo](https://github.com/getodk/docs/issues/new) describing the changes.
-
 1. Follow the guidelines below.
 
 ## The review process
-Bug fixes, pull requests corresponding to issues with a clearly stated goal and pull requests with clear tests and/or process for manual verification are given priority. Pull requests that are unclear or controversial may be tagged as `needs discussion` and/or may take longer to review.
 
-We try to have at least two people review every pull request and we encourage everyone to participate in the review process to get familiar with the code base and help ensure higher quality. Reviewers should ask themselves some or all of the following questions:
```

Comment:
We don't require two people as far as I've seen. Maybe we should add that it requires a review from a committer? I guess this is kind of implicit given a committer is required to merge.